distribution/src/resources/drillbit.sh

```diff
@@ -94,11 +94,24 @@ waitForProcessEnd()
   commandName=$2
   kill_drillbit=$3
   processedAt=`date +%s`
+  triggered_shutdown=false
+  FILE=$DRILL_PID_DIR/.graceful
```

Comment:
I think making it hidden is practical, since this is only serves the purpose of signalling and is not expected to be around visible. 
Also, @dvjyothsna , it might be worth changing the variable name to something like `GRACEFUL_SIGFILE`

Reply:
Changed the name. Should I make it hidden or not?